Xamarin.Forms.Platform.Android/ButtonLayoutManager.cs

```diff
@@ -273,8 +275,8 @@ bool UpdateTextAndImage()
 				return false;
 
 			var textTransform = _element.TextTransform;
-			
-			_renderer.View.SetAllCaps(textTransform == TextTransform.Default);
```

Comment:
I think what makes the most sense here would be to just have a check like this

```C#
          if(label.TextTransform == Forms.TextTransform.Default)
                textView.TransformationMethod = _defaultTransformationMethod;
            else
                 textView.TransformationMethod = null;
```

Reply:
fixed!